test/inductor/test_torchinductor.py

```diff
@@ -2818,7 +2818,8 @@ def test_exp(self):
         def fn(a, b):
             return (torch.exp(a), torch.exp(a + b))
 
-        self.common(fn, (torch.randn(8, 8), torch.randn(8, 8)))
+        # exp(a+b) could be significantly different than exp(a.half()+b.half())
+        self.common(fn, (torch.randn(8, 8), torch.randn(8, 8)), reference_in_float=False)
```

Comment:
Do we want to do this only for MPS, or have `reference_in_float=False` as a global property?

Reply:
Global property, the fact that RNG do not generate those values now, doesn't mean they will not be generated in the future (in my example I demonstrate that test will fail correctness check if one invokes it with those values)